aws/logs_monitoring/template.yaml

```diff
@@ -344,6 +348,8 @@ Resources:
               - Ref: AWS::NoValue
       ReservedConcurrentExecutions:
         Ref: ReservedConcurrency
+      PermissionsBoundary:
+        Ref: PermissionBoundaryArn
```

Comment:
I think you have to create a condition `SetPermissionBoundaryArn` and use `AWS::NoValue` when user does not set `PermissionBoundaryArn` like [this](https://github.com/DataDog/datadog-serverless-functions/pull/209/files#diff-7463e3e489db7e82f7ac591389236943R333), otherwise launching the stack will fail.

Reply:
Thanks for the feedback @tianchu - I have implemented the proposed changes. I also changed parameter name from PermissionBoundary to PermissionsBoundary to be in sync with AWS Naming of the parameter.